katello/katello-backup

```diff
@@ -111,6 +111,8 @@ else
     '/etc/pki/pulp',
     '/etc/pulp',
     '/etc/puppet',
+    '/etc/puppetlabs/puppet/ssl',
+    '/etc/puppetlabs/puppet/foreman_cache_data',
```

Comment:
Probably grab all of /etc/puppetlabs just like we grab all of /etc/puppet.  There's config files in there for agent + server.